pilot/pkg/config/kube/crd/controller.go

```diff
@@ -84,11 +84,16 @@ func NewController(client *Client, options kube.ControllerOptions) model.ConfigS
 	for _, schema := range client.ConfigDescriptor() {
 		out.addInformer(schema, options.WatchedNamespace, options.ResyncPeriod)
 	}
+	go out.queue.Run(options.Stop)
+
+	for _, ctl := range out.kinds {
+		go ctl.informer.Run(options.Stop)
```

Comment:
could this cause a race run informer and queue before calling `RegisterEventHandler`

Reply:
why is that a race?